Analytics/App_Plugins/Analytics/backOffice/OAuthCallback.aspx.cs

```diff
@@ -53,18 +54,17 @@ public partial class OAuthCalllback : UmbracoEnsuredPage {
                 AnalyticsConfig.RefreshToken = refreshToken;
 
                 //Ouput some info about the user
-                Content.Text = "Hi there " + user.Name + ". We have saved your information to a config file, so Umbraco can pull stats from your Analytics account.";
-                Content.Text += "<br /><br />You can now <a href=\"javascript: self.close()\">close this window</a>.";
+                //Using UmbracoUser (obsolete) - somehow it fails to compile when using Security.CurrentUser
+                Content.Text = ui.Text("analytics", "informationSavedMessage", user.Name, UmbracoUser);
```

Comment:
It says that `UmbracoUser` is obsolete and we now should use `Security.CurrentUser` .. however when I use that I get a compile error, when building the project.